main.go

```diff
@@ -42,32 +42,32 @@ var buildRevision = "unknown"
 var buildCompiler = "unknown"
 
 var defaultMetricsPrefix = "ghostunnel"
-var defaultMinTLSVersion = "1.1"
+var defaultMinTLSVersion = "1.2"
 
 var (
 	app = kingpin.New("ghostunnel", "A simple SSL/TLS proxy with mutual authentication for securing non-TLS services.")
 
-	serverCommand        = app.Command("server", "Server mode (TLS listener -> TCP port).")
+	serverCommand        = app.Command("server", "Server mode (TLS listener -> raw TCP/UNIX target).")
```

Comment:
I don't know how I feel about the word "raw":  It makes me think of "raw sockets", which is conflicting with TCP.

Maybe "plain TCP", or just TCP.